example/src/main/java/org/apache/rocketmq/example/benchmark/BatchProducer.java

```diff
@@ -317,7 +318,8 @@ class StatsBenchmarkBatchProducer {
 
     private final LongAdder sendMessageFailedCount = new LongAdder();
 
-    private final Timer timer = new Timer("BenchmarkTimerThread", true);
+    private final ScheduledExecutorService executorService = Executors.newSingleThreadScheduledExecutor(new ThreadFactoryImpl(
+            "BenchmarkTimerThread", Boolean.TRUE));
```

Comment:
The indent here maybe wrong, pls import rmq_codeStyle.xml in style folder and reformat the code

Reply:
AFAIK, if indent is wrong,the checking will not passing